src/lxc/confile.c

```diff
@@ -3101,15 +3102,12 @@ static int parse_line(char *buffer, void *data)
 	value += lxc_char_left_gc(value, strlen(value));
 	value[lxc_char_right_gc(value, strlen(value))] = '\0';
 
-	if (*value == '\'' || *value == '\"') {
-		size_t len;
+	if (*value == '\'' || *value == '\"')
+		value ++;
```

Comment:
Why does the `value ++` become unconditional suddenly?

Reply:
The current codepath works for only that quote marks are in both head and tail like, `'value'` or `"value"` and also must be same value. But somehow if a value of config is as `'value`, `value"` or something like that, then I think we should take care of that cases as well.